ProjectManagmentSystem/package.json

```diff
@@ -26,6 +26,7 @@
     "react-hook-form": "^7.51.4",
     "react-pro-sidebar": "^1.1.0",
     "react-router-dom": "^6.23.0",
+    "react-spinners": "^0.13.8",
```

Comment:
We don't need a package for something like spinner.
We can use this website and get the code for the spinner we want.
https://cssloaders.github.io/